tempesta_fw/ss_skb.h

```diff
@@ -208,4 +208,34 @@ int ss_skb_cutoff_data(SsSkbList *skb_list,
 int ss_skb_process(struct sk_buff *skb, unsigned int *off,
 		   ss_skb_actor_t actor, void *objdata);
 
+/**
+ * While splitting SKB with @skb_fragment we need to lock skb->tail from
+ * being expanded in case of linear part fragmentation tooks place. We use
+ * skb->ipvs_property as a marker as that field is used only for IPVS and
+ * there is no collision with the kernel.
+ *
+ * Note, that in case of using IPVS with the Tempesta this hack MUST be
+ * verified.
```

Comment:
We try to be a good kernel citizenship, while IPVS is very useful thing which is good to integrate with. Probably it's better just to introduce one more flag somewhere near from `skb_page` - we can loose up to 8 bytes, but it's better than break system interoperability.